src/main/java/com/commercetools/sync/categories/README.md

```diff
@@ -0,0 +1,227 @@
+  categorySync.getStatistics().getUpdated(); // 995
+  categorySync.getStatistics().getProcessed(); // 2000
+  categorySync.getStatistics().getReportMessage(); 
+  /*"Summary: 2000 categories were processed in total (1000 created, 995 updated and 5 categories failed to sync)."*/
+   ````
+    
+  <!--- TODO Also add code snippets for building update actions utils! -->
+  
+  Additional configuration for the sync can be configured on the `CategorySyncOptions` instance, according to the need 
+  of the user of the sync:
+  #### `errorCallBack` [Optional]
+  Defines an optional field which represents a callback function which will be called whenever an event occurs
+  that leads to an error alert from the sync process. The function takes two parameters: a `String` error message and a 
+  `Throwable` exception. 
+  
+  An example of setting a callback would be for example logging the error as follows:
+  ````java
+  // instantiate a logger
+  final Logger LOGGER = LoggerFactory.getLogger(myclass.class);
+
+  // instantiating a CategorySyncOptions
+  final CategorySyncOptions categorySyncOptions = CategorySyncOptionsBuilder
+                                                    .of(ctpClient)
+                                                    .setErrorCallBack(LOGGER::error)
+                                                    .build();
+  ````
+  
+  #### `warningCallBack` [Optional]
+  Defines an optional field which represents a callback function which will be called whenever an event occurs
+  that leads to an warning alert from the sync process. The function takes one parameter: a `String` error message.
+  An example of setting a callback would be for example logging the error as follows:
+      
+  ````java
+   // instantiate a logger
+   final Logger LOGGER = LoggerFactory.getLogger(myclass.class);
+    
+   // instantiating a CategorySyncOptions
+   final CategorySyncOptions categorySyncOptions = CategorySyncOptionsBuilder
+                                                        .of(ctpClient)
+                                                        .setWarningCallBack(LOGGER::warn)
+                                                        .build();
+   ````
+  
+  #### `removeOtherLocales` [Optional]
+  Defines an optional field which represents a boolean flag which adds additional localizations without deleting
+  existing ones. If set to true, which is the default value of the option, it deletes the existing localizations. 
+  If set to false, it doesn't delete the existing ones.
+  ````java
+  // instantating options which deletes the existing localizations
+   final CategorySyncOptions categorySyncOptions = CategorySyncOptionsBuilder
+                                                        .of(ctpClient)
+                                                        .setRemoveOtherLocales(true)
+                                                        .build();
+  ````
+  #### `removeOtherSetEntries` [Optional]
```

Comment:
This is removed now from the README. It will be linked to the Integration tests. But do you mean is it in general not clear what this option does to you?